cocos/rendering/custom/layout-graph-editor.ts

```diff
@@ -662,6 +671,31 @@ export class LayoutGraphInfo {
             }
         }
     }
+    private buildGroupBlocks (visDB: VisibilityDB, rate: UpdateFrequency, blocks: EffectAsset.IBlockInfo[], db: DescriptorDB, counter: DescriptorCounter): void {
+        const visBlock = visDB.getBlock({
+            updateFrequency: rate,
+            parameterType: ParameterType.TABLE,
+            descriptorType: DescriptorTypeOrder.UNIFORM_BUFFER,
+        });
+        for (const info of blocks) {
```

Comment:
Can avoid using `const ... of`

Reply:
I will change the for loops.
The `layout-graph-editor.ts` is only used in editor and not in runtime. It will not be packed.

Reply:
ok